README.md

```diff
@@ -33,15 +34,25 @@ and
 *or if you're using Gradle:*
 
 ```bash
-implementation 'com.rudderstack.sdk.java:rudderanalytics-client:1.0.0'
+implementation 'com.rudderstack.sdk.java:rudderanalytics-client:1.0.1'
 ```
 
 ## Initialize ```RudderClient```
 ```java
+FlushBlock flushBlock = FlushBlock.create(); // Initialization in order to block further method invocation until the flush completes. 
 RudderAnalytics analytics = RudderAnalytics.builder(
         "write_key",
         "http://data-plane-url"
-).build();
+)
+.plugin(flushBlock.plugin()) // It is needed to use FlushBlock
+.plugin(new PluginLog()) // Specific Logging for FlushBlock
+.build();
+
+...YOUR CODE...
+
+analytics.flush(); // Trigger a flush.
+flushBlock.block(); // Block until the flush completes.
```

Comment:
Let's add two method signatures here:

`analytics.flush()` and `analytics.blockingFlush()`

The first one will do it async and in the second one it will call `flushBlock.block()` internally